src/main/java/org/wildfly/security/sasl/scram/ScramSaslClient.java

```diff
@@ -66,6 +67,8 @@
     private byte[] saltedPassword;
     private byte[] serverFirstMessage;
 
+    private static final Logger log = Logger.getLogger(ScramSaslServer.class);
```

Comment:
You should not select a log category by class name, but rather by "process".  Better would be ``Logger.getLogger("org.wildfly.security.sasl.scram")`` or even ``Logger.getLogger("org.wildfly.security.sasl.scram.server")``.